components/brave_wallet/browser/brave_wallet_p3a.h

```diff
@@ -92,6 +92,7 @@ class BraveWalletP3A : public mojom::BraveWalletServiceObserver,
   void OnNetworkListChanged() override {}
   void OnDiscoverAssetsCompleted(
       std::vector<mojom::BlockchainTokenPtr> discovered_assets) override {}
+  void OnResetWallet() override {}
```

Comment:
Can we use new BraveWalletServiceObserverBase here then and remove redundand overrides?

Reply:
Actually applies to all usages of brave_wallet::mojom::BraveWalletServiceObserver

Reply:
Done